src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -96,22 +122,22 @@
                                 </DataTemplate>
                             </ComboBox.ItemTemplate>
                         </ComboBox>
-                        <TextBox Width="56"
-                                 TextWrapping="Wrap"
-                                 Style="{StaticResource DisabledWhenUnselectedTextBoxStyle}"
-                                 AutomationProperties.Name="{x:Static p:Resources.Width}"
-                                 Margin="8,0,0,0">
-                            <TextBox.Text>
-                                <Binding Converter="{StaticResource AutoDoubleConverter}"
-                                         Path="Width"
-                                         UpdateSourceTrigger="PropertyChanged">
-                                    <Binding.ValidationRules>
-                                        <local:AutoDoubleValidationRule/>
-                                    </Binding.ValidationRules>
-                                </Binding>
-                            </TextBox.Text>
-                        </TextBox>
-                        <TextBlock VerticalAlignment="Center"
+                            <ui:NumberBox SpinButtonPlacementMode="Compact"
```

Comment:
After switching from `TextBox` to `NumberBox`, we don't need anymore the `AutoDoubleConverter` and the `AutoDoubleValidationRule`.
It should be enough to bind the control directly to the property.
In case I can ask someone on my team to to make the change.

Reply:
Also, the `NumberBox` uses the `Value` field, not the `Text` field.

Reply:
@enricogior Thanks! Swapped out the Text for value and removed the AutoDoubleValidationRule, since that's no longer required - but not 100% sure, so please advise :).